Add google drive section (#20480)
fleetdm/fleet: handbook/company/communications.md

## Levels of confidentiality

Fleet uses these levels to standardize a commitment to minimal esotericism across the company.  
- **Public:**   _Share with anyone, anywhere in the world_
- **Confidential:**  _Share only with team members who've signed an NDA, consulting agreement, or employment agreement_
- **Classified:**  _Share only with founders of Fleet, business operations, and/or the people involved.  e.g., US social security numbers during hiring_

### Document titles

Fleet uses these levels to standardize a commitment to minimal esotericism across the company. 
- **"Public":** _(Available to public)_ 
- _(Confidential - for Fleet eyes only)_     
- **"¶":** _(E-group - Direct reports the the CEO)_
- **"¶¶":** _(Classified - CEO, Apprentice, and BizOps)_
- **"¶¶¶":** _(CEO, Apprentice to the CEO, and board members)_

## Google Drive

> Fleet uses Google Drive in order to keep files correctly bucketed and as accesable as possible for all team members. Here are some best practices for to keep in mind:

Reviewer: ```suggestion
Fleet uses Google Drive to keep files correctly bucketed and accessible for team members. When adding a file, you can: 
```

> - Check for existing files defore uploading to avoid duplication.

Reviewer: ```suggestion
- Check for existing files to avoid duplication.
```

> - Avoid breaking google drive links when uploading new versions of existing files.  In most cases, you can update files by using the "Manage versions" option (in the right click menu) so links continue to work.

Reviewer: ```suggestion
- Avoid breaking Google Drive links. In some cases, you can update files by using the "Manage versions" option (in the right-click menu) so that links continue to work.
```
